modules/installing-gcp-user-defined-labels-and-tags.adoc

```diff
@@ -0,0 +1,74 @@
+// Module included in the following assemblies:
+// * installing/installing_gcp/installing-gcp-customizations.adoc
+
+:_content-type: CONCEPT
+[id="installing-gcp-user-defined-labels-and-tags_{context}"]
+= User-defined labels and tags for GCP
+
+
+Google Cloud Platform (GCP) provides labels and tags that help to identify and organize the resources created for a specific {product-title} cluster, making them easier to manage. 
+
+You can define labels and tags for each GCP resource only during {product-title} cluster installation. 
+
+[IMPORTANT]
+====
+User-defined labels and tags are not supported for {product-title} clusters upgraded to {product-title} 4.14 version.
+====
+
+.User-defined labels
+
+User-defined labels and {product-title} specific labels are applied only to resources created by {product-title} installer and its core components such as: 
+
+* GCP filestore CSI Driver Operator
+* GCP PD CSI Driver Operator
+* Image Registry Operator
+* Machine API provider for GCP
+
+
+The user-defined labels and {product-title} specific labels are not applied on the resources created by any other operators or the kubernetes in-tree components that create resources, for example, the Ingress load balancers.
```

Comment:
I think this is also a limitation. Kindly confirm.

Reply:
This was more like intro but the info was added in the limitations and then to avoid duplication of the data, I had removed it. Let me change it.

Reply:
Confirmed from the SME "It's not a limitation, since those components are not owned by openshift, it's out of scope."